x/ibc/09-localhost/types/msgs.go

```diff
@@ -65,3 +65,10 @@ func (msg MsgCreateClient) GetClientType() string {
 func (msg MsgCreateClient) GetConsensusState() clientexported.ConsensusState {
 	return nil
 }
+
+// InitializeClientState implements clientexported.MsgCreateClient
+// localhost is a special case that require the running chain's context to initialize
+// the client state, thus this function is a no-op
+func (msg MsgCreateClient) InitializeClientState() clientexported.ClientState {
```

Comment:
can we add a test on 02-client with the localhost client to ensure it doesn't panic?